src/main/java/racing/InputView.java

```diff
@@ -0,0 +1,30 @@
+package racing;
+
+import racing.message.ViewMessage;
+
+import java.util.Scanner;
+
+public class InputView {
+    private int carNumber;
+    private int tryCount;
+
+    public InputView() {
+        System.out.println(ViewMessage.START_MESSAGE_CAR_NUMBER.getMessage());
+        this.carNumber = InputValue();
+        System.out.println(ViewMessage.START_MESSAGE_TRY_COUNT.getMessage());
+        this.tryCount = InputValue();
+    }
+
+    public int InputValue() {
```

Comment:
메서드명이 대문자로 시작하고 있네요 :)
```suggestion
    public int inputValue() {
```

Reply:
수정했습니다. 감사합니다.